src/Introspection.cpp

```diff
@@ -19,6 +20,7 @@ using std::map;
 
 namespace Halide {
 namespace Internal {
+namespace Introspection {
 
 #ifdef __APPLE__
```

Comment:
This seems to be OSX-and-glibc only. Do we need alternate paths for Win, Android, etc?